unittest/python/test_rk4_comp.py

```diff
@@ -0,0 +1,168 @@
+import numpy as np
+
+import pinocchio
+import example_robot_data
+
+import crocoddyl
+import crocoddyl_legacy
+
+
+class SimpleQuadrotorProblem:
```

Comment:
Simplify the unittest. Let me guide you:
  1. Create a base class for the unitest where you compare both problems (e.g. https://github.com/loco-3d/crocoddyl/blob/devel/unittest/bindings/test_actions.py#L14)
  2. Define the different problems (i.e. quadrotor and arm) in a derived class (e.g. https://github.com/loco-3d/crocoddyl/blob/devel/unittest/bindings/test_actions.py#L67)
 3. List the test (e.g. https://github.com/loco-3d/crocoddyl/blob/devel/unittest/bindings/test_actions.py#L132)

Reply:
done